src/__tests__/linter.test.ts

```diff
@@ -655,24 +654,6 @@ responses:: !!foo
     ]);
   });
 
-  test('should report invalid schema $refs', async () => {
```

Comment:
@P0lip We're dropping this test in favor of a more focused one targeting the `schema` function. Do we have an existing end to end test that shows how our own resolver errors when it cannot How about adding one test that would put under the light how our own resolver errors when it cannot properly resolve a ref in `resolved` context?

Reply:
@P0lip Thoughts?

Reply:
Yep, will do.

Reply:
Done!